inference-engine/src/gna_plugin/gna_device.cpp

```diff
@@ -376,6 +378,11 @@ void GNADeviceHelper::close() {
     GNADeviceClose(nGNAHandle);
     nGNAHandle = 0;
 #else
+    if (wasAnyRequestScheduled) {
+        try {
+            wait(lastSheduledRequestId);
```

Comment:
1) this is release branch so #2401 will never be merged
2) this review will most likely be closed since there is high chance that no additional round of commits will go to release. Master branch was merged already with slightly different solution (https://github.com/openvinotoolkit/openvino/pull/2336)
3) as for verifying the status, the most you could do would be to log that the status was unsuccessfull, since it is destructor you cannot really have any extensive logic here. What behavior would you want in case of not successfull execution? Propose something please.